Hbm.Automation.API/Weighing/WTX/WTXJet.cs

```diff
@@ -858,12 +858,14 @@ public override void CalculateAdjustment(double scaleZeroLoad_mVV, double scaleC
         ///<inheritdoc/>
         public override bool AdjustZeroSignal()
         {
-            Connection.WriteInteger(JetBusCommands.CIA461ScaleCommand, SCALE_COMMAND_CALIBRATE_ZERO); 
+            Connection.WriteInteger(JetBusCommands.CIA461ScaleCommand, SCALE_COMMAND_CALIBRATE_ZERO);
 
-            while (Convert.ToInt32(Connection.ReadFromBuffer(JetBusCommands.CIA461ScaleCommandStatus)) != SCALE_COMMAND_STATUS_ONGOING)
+            int com = Convert.ToInt32(Connection.ReadFromBuffer(JetBusCommands.CIA461ScaleCommandStatus));
+            while ((com != SCALE_COMMAND_STATUS_ONGOING) && (com != SCALE_COMMAND_STATUS_OK))
```

Comment:
Hi Roland,
How is that supposed to work?
The loop will only break, if status is not ONGOING and not OK ???
Think about a minimum delay before entering the ONGOING loop!
BR
Dirk

Reply:
Hi Dirk,
I'm sorry but I think you got your logic backwards.
Actually the loop will **continue** if the status is neither ONGOING nor OK.
Meaning it will break if the status reaches either ONGOING or OK.

Again, sorry for replying to a three year old issue. I just couldn't help but notice.